mixer/pkg/loadshedding/throttler.go

```diff
@@ -15,13 +15,10 @@
 package loadshedding
 
 import (
-	"context"
 	"fmt"
 
-	"go.opencensus.io/stats"
-	"go.opencensus.io/stats/view"
-
 	"istio.io/pkg/log"
+	"istio.io/pkg/monitoring"
```

Comment:
@douglas-reid looks like this dependency fails on 1.2. Can you confirm?

Reply:
@fpesce yes, this cherry-pick will require some modifications before it works on 1.2. I'll take a stab today, if i can manage it.

Reply:
ok. @fpesce i think i've removed the refs to the non-existent package in 1.2

Reply:
Thanks @douglas-reid !